src/olympia/devhub/templates/devhub/new-landing/index.html

```diff
@@ -1 +1,70 @@
-New DevHub Landing
+<!DOCTYPE html>
+<html lang="{{ LANG }}" dir="{{ DIR }}">
+  <head>
+    <meta charset="utf-8">
+      {% if not settings.ENGAGE_ROBOTS %}
+        <meta name="robots" content="noindex">
+      {% endif %}
+    {% block extrameta %}{% endblock %}
+    <title>{% block title %}{{ dev_page_title() }}{% endblock %}</title>
+
+    <link rel="shortcut icon" type="image/x-icon"
+          href="{{ STATIC_URL }}img/favicon.ico">
+
+    {% block rss_feed %}{% endblock %}
+
+    {# L10n: {0} is an application, like Firefox. #}
+    <link title="{{ _('{0} Add-ons')|f(APP.pretty) }}"
+          rel="search" type="application/opensearchdescription+xml"
+          href="{{ url('amo.opensearch') }}" />
+
+    {% block site_css %}
+      {{ css('devhub/css') }}
+    {% endblock %}
+
+    {% block extrahead %}{% endblock %}
+
+    {% if request.user.is_authenticated() %}
+      <meta name="csrf" content="{{ csrf_token }}">
+    {% endif %}
+
+    {{ js('preload') }}
+  </head>
+  <body class="html-{{ DIR }} {{ request.APP.short }} {% block bodyclass %}{% endblock %}"
+        data-app="{{ request.APP.short }}"
```

Comment:
Good point, I removed them all except for raven (since we do load some JS, generically), a9f818d